feat: First draft for clisk contributor documentation (#282)
cozy/cozy.github.io: src/tutorials/CliSK/6 - Contribution/6.1 - Deploy my konnector on the registry.md

#### Validation and release

###### Existing konnector modification or corrections

To validate your modifications on an existing konnector, you must have made a GitHub Pull Request on the master of the original repo in our [konnector's organisation](https://github.com/konnectors) . We will review your work, maybe ask for clarifications, little changes if needed and we will test it. Then when everything looks good, we'll merge your branch into the master and release a new version with your modifications. We will handle everything after the merge, but you will still be the owner of those modifications commits.

###### New konnector

> If you had built a konnector that don't exist yet in the cozy library, you should submit it through [the contributor communication channel - TODO]() . Once tested on our side, you will need to transfer it to the `konnectors` organisation. That way it can be part of the continuous deployment chain, with CI, libraries updates etc ... we will do the necessary to make it available in the application store so anyone can install it.

Reviewer: ```suggestion
If you have built a konnector that doesn't exist yet in the Cozy library, you should submit it through [the contributor communication channel - TODO]() . Once tested on our side, you will need to transfer it to the `konnectors` organization. This way it can be part of the continuous deployment chain, with CI, library updates etc. We will do what is necessary to make it available in the application store so anyone can install it.
```
